7.1/contrib/services.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# https://medium.com/@tomahock/passing-system-environment-variables-to-php-fpm-when-using-nginx-a70045370fad
+printf "\nclear_env = no\n" >> /etc/php-fpm.d/www.conf
+# remote empty values then rewrite as env[key] = 'value'
+env | awk -F '=' '$2 !~ /^$/{print}'  | sed "s/\([^=]*\)=\(.*\)/env[\1] = '\2'/" >> /etc/php-fpm.d/www.conf
+printf "\n" >> /etc/php-fpm.d/www.conf
```

Comment:
This copies the env of the pod into the environment variables set on the php-fpm workers. Without that you cannot have openshift configure php using environment variables.